### PR TITLE
Fix v0.0.5

### PR DESCRIPTION
**Pull request by Jan-PieterK**
bug fix first segment "|" not working correctly. Now it works.

Now when you upload a HL7 message the descriptions are correct.

### CODE REVIEW

File: hl7/_from_hl7.py [hunk 1]
```diff
@@ -93,8 +99,13 @@ def _prepare_csv_content(csv_content: str) -> str:
     ).replace(_FIELD_SEPARATOR_FIELD, FIELD_SEPARATOR)
 
 
-def _get_description(segment_name: str, index1: int, index2) -> str:
-    key_json = f"{segment_name}.{index1}.{index2}"
+def _get_description(segment_name: str, index1: int, index2=None) -> str:
```
Comment (zEktONO) [inline hl7/_from_hl7.py:102]: You need to add type annotation here.

File: hl7/_from_hl7.py [hunk 2]
```diff
@@ -70,7 +70,13 @@ def _convert(hl7_content: str) -> Iterator[Tuple[str, str, str, str]]:
             for index2, field2 in enumerate(field.split("^"), start=1):
                 if not field2:
                     continue
-                description = _get_description(segment_name, index1, index2)
+                if index2 == 1:
```
Comment (zEktONO) [inline hl7/_from_hl7.py:73]: Do we also want to do this for `index3` in the next function?

For instance: `3.5.1` becomes `3.5`?

File: pyproject.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "py-hl7"
-version = "0.0.2"
+version = "0.0.3"
```
Comment (zEktONO) [inline pyproject.toml:3]: Shouldn't this be 0.0.5? (Name of this PR)

File: hl7/_from_hl7.py [hunk 4]
```diff
@@ -70,7 +70,13 @@ def _convert(hl7_content: str) -> Iterator[Tuple[str, str, str, str]]:
...
+                   index2 = ""
+                   index1 = int(index1)
```
Comment (zEktONO) [inline hl7/_from_hl7.py:75]: This doesn't need a variable override. You can directly add `int(index1)` as an argument to the `_get_description` function.

**Review by Zektono — CHANGES_REQUESTED** [review]

**Review by Zektono — APPROVED** [review]

**Pull request merged**